ooni/errors.py

```diff
@@ -6,7 +6,7 @@
 from twisted.internet.error import ConnectionRefusedError, TCPTimedOutError
 from twisted.internet.error import DNSLookupError, ConnectError, ConnectionLost
 from twisted.internet.error import TimeoutError as GenericTimeoutError
-from twisted.internet.error import ProcessDone
+from twisted.internet.error import ProcessDone, ConnectionDone
```

Comment:
Responding to myself, this probably indicates that the proxy closed the connection.